client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala

```diff
@@ -1100,10 +1180,10 @@ object DeltaSharingRestClient extends Logging {
   val RESPONSE_FORMAT = "responseformat"
   val READER_FEATURES = "readerfeatures"
   val DELTA_SHARING_CAPABILITIES_ASYNC_READ = "asyncquery"
+  val DELTA_SHARING_CAPABILITIES_INCLUDE_END_STREAM_ACTION = "includeendstreamaction"
```

Comment:
Should we simplify and call this: endstreamaction?

I think include is implied; otherwise ideally the client should send include* and server should send included*

Reply:
updated.